cunumeric/module.py

```diff
@@ -1805,7 +1805,11 @@ def concatenate(
 
     # flatten arrays if axis == None and concatenate arrays on the first axis
     if axis is None:
-        inputs = list(inp.ravel() for inp in inputs)
+        # Reshape arrays in the `array_list` to handle scalars
+        reshaped = _atleast_nd(1, tuple(inputs))
```

Comment:
I don't think you need to call `tuple(inputs)`, you could just do `_atleast_nd(1, inputs)`, and similar in other places in this PR where you call `_atleast_nd(1, tuple(tup))`. You may need to extend the type signature of `_atleast_nd` to take `ndim: int, arys: Sequence[ndarray, ...]`.

Reply:
Fixed. Thanks.